common/src/main/java/com/wynntils/core/webapi/WebManager.java

```diff
@@ -118,6 +119,9 @@ public static void reset() {
         // tryLoadTerritories
         territories.clear();
 
+        // tryLoadMaps
```

Comment:
Why this?

(Please don't mix different bug fixes in a separate PR, it makes for a messy history, messy reverts in case of failures, and all kind of confusion)

Reply:
Will do from now on